cmd/influx/write.go

```diff
@@ -52,6 +53,8 @@ type writeFlagsBuilder struct {
 	Encoding                   string
 	ErrorsFile                 string
 	RateLimit                  string
+	Compressed                 bool
```

Comment:
I think this needs to become `*bool` so we can distinguish not-set from a true `false`

Reply:
Made it a `string` instead to match the existing input-format option